pkg/crypto/keys/internal/scrypt/scrypt.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2012 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package scrypt implements the scrypt key derivation function as defined in
+// Colin Percival's paper "Stronger Key Derivation via Sequential Memory-Hard
+// Functions" (https://www.tarsnap.com/scrypt/scrypt.pdf).
+package scrypt
```

Comment:
I'm not a fan of this change, but if `x/crypto` is not expected to be modified, then OK.

Reply:
It's a pretty stable package. Yeah, I also don't like having this in our codebase. Yes, I have tried upgrading NeoFS SDK to go-1.24 version of NeoGo and x/crypto is still in dependencies. But I think that's because of circular dependency between SDK and NeoGo, since NeoGo still imports an old version the dependency remains. Once both are updated I expect x/crypto to be gone, but if that's not the case, reverting from internal scrypt to x/crypto is trivial.